caffe2/opt/optimizer.cc

```diff
@@ -0,0 +1,48 @@
+#include "caffe2/opt/optimizer.h"
+
+#include "caffe2/opt/converter.h"
+#include "caffe2/opt/mobile.h"
+#include "caffe2/opt/fusion.h"
+
+namespace caffe2 {
+namespace opt {
+
+void workspaceOptimizations(nom::repr::NNModule* nn, Workspace* ws, int level) {
+  switch (level) {
+    case 1:
+      opt::fuseConvBN(nn, ws);
+    case 0:
+    default:
+      break;
+  }
+}
+
+void graphOptimzations(nom::repr::NNModule* nn, int level) {
```

Comment:
I don't like the level idea (why would you want *a little of* optimizations?), but if you're using it, check for `level >= 1`, not `level == 1`

Reply:
this mimics gcc/clang which provide fast to start optimizations and optimizations that may take a while to process

Reply:
But `level = 2` should also include `level = 1` optimizations, right? It does in gcc/clang.